trimesh/voxel.py

```diff
@@ -266,6 +291,62 @@ def voxelize_subdivide(mesh, pitch, max_iter=10):
 
     return voxels_sparse, origin_position
 
+def voxelize_subdivide_solid(mesh,pitch,max_iter=10):
+    voxels_sparse,origin_position=voxelize_subdivide(mesh,pitch,max_iter)
```

Comment:
If we passed the sparse voxels, objects could re- use that calculation, something like:

```
def voxelize_subdivide_solid(sparse, sparse_origin):
```